commands/roll.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (bb8) {
+    console.log('Let\'s Roll!!');
+
+    setInterval(function () {
+        var direction = Math.floor(Math.random() * 360);
+        bb8.roll(150, direction);
```

Comment:
This reminds me of doing [Logo!](http://www.bbc.co.uk/schools/gcsebitesize/ict/measurecontrol/1logocontrolrev1.shtml)